test/db/organizations.csv

```diff
@@ -1,3 +1,3 @@
-_id,name,created,isdeleted,additionalinformation
-'5ab0c6a674d0af7b003b23ac','Organization_X','lorem ipsum','2018-03-20 09:30:31.91+01',f
-'6bb0c6a674d0af7b003b23bd','Organization_Y','foo bar','2018-03-24 09:30:31.91+01',f
+_id,name,additionalinformation,logoUrl,created,isdeleted
+'5ab0c6a674d0af7b003b23ac','Organization_X','lorem ipsum','abc','2018-03-20 09:30:31.91+01',f
+'6bb0c6a674d0af7b003b23bd','Organization_Y','foo bar','def','2018-03-24 09:30:31.91+01',f
```

Comment:
If we use the testDB locally, then it would try to load the image 'def' which does not exist. Maybe we should set the logourl to "/assets/images/mpi-logos.svg". 
But this is only a really small problem (because we are currently unable to login as a test-user anyways).